test/atm/global-workflow/gdas_prototype.yaml.j2

```diff
@@ -0,0 +1,6 @@
+observers:
+{% filter indent(width=2) %}
+{% include 'atm/obs/config/amsua_n19.yaml.j2' %}
+{% include 'atm/obs/config/sondes.yaml.j2' %}
+{% set cycle = current_cycle | strftime("t%Hz") %}
```

Comment:
```suggestion
```
Remove `cycle` as it is not used in the template.

Reply:
Change committed at b174bf2